src/main/java/com/flightstats/hub/dao/aws/S3Config.java

```diff
@@ -130,26 +137,23 @@ private void updateMaxItems(ChannelConfig config, ContentKey latest) {
             keys.addAll(channelService.query(query));
             if (keys.size() == config.getMaxItems()) {
                 ContentKey limitKey = keys.first();
-                logger.info("deleting keys before {}", limitKey);
+                log.info("deleting keys before {}", limitKey);
                 channelService.deleteBefore(name, limitKey);
             }
         }
 
         private void updateTtlDays() {
-            logger.info("updateTtlDays");
+            log.info("updateTtlDays");
             ActiveTraces.start("S3Config.updateTtlDays");
-            int maxRules = HubProperties.getProperty("s3.maxRules", 1000);
+            int maxRules = HubProperties.getProperty("s3.maxRules", S3_LIFECYCLE_RULES_AVAILABLE);
             List<BucketLifecycleConfiguration.Rule> rules = new ArrayList<>();
-            if (maxRules == 0) {
-                rules.add(new BucketLifecycleConfiguration.Rule()
-                        .withId("OneDay")
-                        .withExpirationInDays(1)
-                        .withStatus(BucketLifecycleConfiguration.ENABLED));
-            } else {
+
+            if (maxRules > 0 && maxRules <= S3_LIFECYCLE_RULES_AVAILABLE) {
```

Comment:
I don't have the best understanding of these rule configurations, so just asking:
If this statement is false, we do nothing, what happens then? We have no rules apportioned? Or is there an automatic default?

Reply:
if s3.maxRules property is not available, default value is taken (990)